content/tutorials/get-started/get-started-using-python.md

```diff
@@ -0,0 +1,215 @@
+
+##### Prepare
+
+To prepare the transaction:
+
+```py
+from xrpl.models.transactions import Payment
+
+my_tx_payment = Payment(
+    account=test_wallet.classic_address,
+    amount="2200", # in drops: https://xrpl.org/basic-data-types.html#specifying-currency-amounts
+    destination="rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe",
+    sequence=test_wallet.next_sequence_num,
+)
+```
+
+##### Sign
+
+To sign the transaction:
+
+```py
+from xrpl.transaction import send_reliable_submission
+
+my_tx_payment_signed = safe_sign_transaction(my_tx_payment,test_wallet)
+```
+##### Send 
+
+To send the transaction:
+
+```py
+from xrpl.transaction import send_reliable_submission
+
+tx_response = send_reliable_submission(my_tx_payment, test_wallet, client)
+```
+
+##### Derive an X-address
+
+You can use `xrpl-py`'s [`xrpl.core.addresscodec`](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.core.addresscodec.html) module to derive an [X-address](https://xrpaddress.info/) from the `Wallet.classic_address` field:
+
+
+{{ include_code("_code-samples/xrpl-py/get-acct-info.py", start_with="# Derive an x-address from the classic address:", end_before="# Look up info about your account", language="py") }}
+
+The X-address format [packs the address and destination tag](https://github.com/xrp-community/standards-drafts/issues/6) into one more user-friendly value. 
+
+### {{n.next()}}. Query
+
+You can query the XRP Ledger to get information about [a specific account](account-methods.html), [a specific transaction](tx.html), the state of a [current or a historical ledger](ledger-methods.html), and [the XRP Ledger's decentralized exhange](path-and-order-book-methods.html). You need to make these queries, among other reasons, to look up account info to follow best practices for [reliable transaction submission](reliable-transaction-submission.html).  
+
+Here, we'll use `xrpl-py`'s [`xrpl.account`](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.account.html) module to look up information about the [wallet we generated](#generate-wallet) in the previous step. 
+
+
+{{ include_code("_code-samples/xrpl-py/get-acct-info.py", start_with="# Look up info about your account", language="py")  }}
+
+
+
+### {{n.next()}}. Putting it all together
+
+Using these building blocks, we can create a simple Python app that:
+
+1. Generates a wallet on the Testnet.
+2. Connects to the XRP Ledger. 
+3. Looks up and prints information about the account you created. 
+
+
+```python
+{% include '_code-samples/xrpl-py/get-acct-info.py' %}
+```
+
+To run the app, you can copy and paste the code into an editor or IDE and run it from there. Or you could download the file from the [XRP Ledger Dev Portal repo](https://github.com/ripple/ripple-dev-portal/tree/master/content/_code-samples/xrpl-py) and run it locally:
+
+
+```sh
+git clone git@github.com:ripple/xrpl-dev-portal.git
+cd path/to/xrpl-dev-portal/content/_code-samples/xrpl-py/get-acct-info.py
+python3 get-acct-info.py 
```

Comment:
Hmm, `wget` is not installed by default on macos. Easy enough to install via `brew` but that seems like a lot of steps to get one file. 

You could also get the content via `curl`:

```
curl -0  "https://raw.githubusercontent.com/ripple/xrpl-dev-portal/xrpl-py-tutorial/content/_code-samples/xrpl-py/get-acct-info.py"
```

But that only writes it to `stdout`. I think you could also tweak that to write to a specific file locally but I haven't been able to get that to work.

Reply:
`git` isn't installed by default on macOS, either, though you're right that git is more likely to be useful for other things anyway

Reply:
Since the xrpl-dev-portal repo will get cloned into a folder of the same name relative to where you are when you run the `git clone` command, the following command should consistently work:

```suggestion
cd xrpl-dev-portal/content/_code-samples/xrpl-py/get-acct-info.py
```